task-management/package-lock.json

```diff
@@ -0,0 +1,13664 @@
+{
```

Comment:
Do not commit this file , please remove it from the PR

Reply:
done